input/exercises/solutions/cs/test_basic_2018a.md

```diff
@@ -0,0 +1,50 @@
+title: Test základních znalostí 2018a
+--- 
+# Zatmění čtvercového měsíce
+ ``` matlab
+	  A=imread('kytka256.jpg');
+    vektor=uint8([0:2:255 255:-2:0]);
+    plocha = repmat(vektor,256,1);
```

Comment:
Hodil by se nějaký komentář. Podobně je to na více místech. Tam kde to nemusí být uplně jasné co se děje, patří komentář.